app/controllers/secrets_controller.rb

```diff
@@ -117,7 +115,7 @@ def error_info
     # want more accurate 'Forbidden'.
     {
       success: false,
-      error_message: (resource_visible? ? $ERROR_INFO.message : 'Forbidden')
+      error_message: (!resource_visible? ? $ERROR_INFO.message : 'Forbidden')
```

Comment:
Why not write this as `resource_visible? ? 'Forbidden' :  $ERROR_INFO.message` ? In addition, I believe the logic here is correct but is confusing given the comment written above.

Reply:
Fixed on latest change